Terminal.Gui/Text/TextFormatter.cs

```diff
@@ -546,6 +570,7 @@ public static string Justify (string text, int width, char spaceChar = ' ', Text
 		///  If <see langword="false"/>, trailing spaces at the end of wrapped lines will be trimmed.</param>
 		/// <param name="tabWidth">The number of columns used for a tab.</param>
 		/// <param name="textDirection">The text direction.</param>
+		/// <param name="multiLine">If <see langword="true"/> new lines are allowed.</param>
```

Comment:
What happens if it's false and newlines are included?

Reply:
I updated the documents. If it's not correct please let me know, please.